client/src/components/CreateMeetup/LocationPicker.js

```diff
@@ -0,0 +1,211 @@
+import React, { useEffect, useMemo, useRef, useState } from "react";
+import { Grid, TextField, Typography } from "@material-ui/core";
+import LocationOnIcon from "@material-ui/icons/LocationOn";
+import Autocomplete from "@material-ui/lab/Autocomplete";
+import parse from "autosuggest-highlight/parse";
+import throttle from "lodash/throttle";
+import PropTypes from "prop-types";
+
+const googleMapsKey = ""; // fill this in with your own key
```

Comment:
Contact me for the key, if you need it.

Maybe, we could use an environment variable. Hmm. https://create-react-app.dev/docs/adding-custom-environment-variables/